build/BUILD.dawn

```diff
@@ -322,6 +322,7 @@ DAWN_COMMON_SRCS = [
     "src/dawn/common/CoreFoundationRef.h",
     "src/dawn/common/DynamicLib.cpp",
     "src/dawn/common/DynamicLib.h",
+    "src/dawn/common/Enumerator.h",
```

Comment:
@edevil we really need to figure out not to duplicate all this build configuration but use it directly